getting-started/initialize-the-sdk.md

```diff
@@ -416,78 +416,105 @@ To ensure accurate session and crash reporting, this call must be added to every
+
+```objectivec
+- (void) applicationWillEnterForeground:(UIApplication *)application {
+    [ACPCore lifecycleStart:nil];
+}
+```
+
+When the app enters the background, pause Lifecycle data collection from your app's `applicationDidEnterBackground:` delegate method:
 
 ```objectivec
- // Objective-C
  - (void) applicationDidEnterBackground:(UIApplication *)application {
-     [ACPCore lifecyclePause];
+    [ACPCore lifecyclePause];
  }
 ```
 
 #### Swift
 
-In swift, ACPCore includes ACPLifecycle :
+In swift, importing `ACPCore` also imports the necessary Lifecycle APIs:
 
 ```swift
 import ACPCore
 ```
 
-Register the framework with Mobile Core by adding the following in your app's `didFinishLaunchingWithOptions`:
+Register the Lifecycle extension with the SDK Core by adding the following in your app's `application:didFinishLaunchingWithOptions:` delegate method:
 
 ```swift
 func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-ACPLifecycle.registerExtension();
-​  // Override point for customization after application launch.
-  return true;
+    // register the lifecycle extension
+    ACPLifecycle.registerExtension();
 }
 ```
 
-Start Lifecycle data collection by adding `lifecycleStart` to your app's`didFinishLaunchingWithOptions`
+Start Lifecycle data collection by calling `lifecycleStart:` from within the callback of the `ACPCore::start:` method in your app's `application:didFinishLaunchingWithOptions:` delegate method.
+
+{% hint style="warning" %} If your iOS application supports background capabilities, your `application:didFinishLaunchingWithOptions:` method may be called when iOS launches your app in the background. If you do not want background launches to count towards your lifecycle metrics, then `lifecycleStart:` should only be called when the application state is not equal to `UIApplicationStateBackground`. {% endhint %}
 
 ```swift
-// Swift
 func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
-ACPCore.start {
-    ACPCore.lifecycleStart(nil);
+    // register the lifecycle extension
+    ACPLifecycle.registerExtension();
+
+    let appState = application.applicationState;            
+    ACPCore.start {
+        // only start lifecycle if the application is not in the background    
+        if appState != .background {
+            ACPCore.lifecycleStart(nil)
+        }    
+    }    
 }
-    return true
+```
+
+When launched, if your app is resuming from a backgrounded state, iOS may call your `applicationWillEnterForeground:` delegate method. We also need to call `lifecycleStart:` here, but this time we do not need all of the supporting code we used in `application:didFinishLaunchingWithOptions:`:
```

Comment:
- Please change "may" to "might".

- Please reword the 2nd sentence so that it reads as follows:

"You also need to call `lifecycleStart:, but this time you do not need all of the supporting code that you used in application:didFinishLaunchingWithOptions::."